src/CustomError.js

```diff
@@ -0,0 +1,9 @@
+class CustomError extends Error {
+  static PREFIX = '[ERROR]';
+
+  constructor(message) {
+    super(`${CustomError.PREFIX} ${message}`);
+  }
+}
+
+export default CustomError;
```

Comment:
```suggestion
class CustomError extends Error {
  static PREFIX = '[ERROR]';
  constructor(message, name) {
    super(`${CustomError.PREFIX} ${message}`);
    this.name = name;
  }
}

export default CustomError;
```

저는 CustomError 공부할 때 name 넘겨주는 것만 많이 봤는데 PREFIX까지 넣어주니 훨씬 좋네요! 위와 같이 사용해주면 좋을 것 같아요 배워갑니다:)

Reply:
해당 방식을 사용하면 OutputView에서 error의 name까지 찾을 수 있겠군요~
자세한 조언까지 첨부해주셔서 감사합니다! 👍